pkg/sat/loader_test.go

```diff
@@ -190,7 +189,7 @@ func TestLoader_Load(t *testing.T) {
 			expectedPackages(g, model, map[string][]string{
 				"A": []string{"0:1.0-1"},
 			})
-			expectedVars(g, model, "A-0:1.0-1(A)")
+			expectedVars(g, model, "A-0:1.0-1")
```

Comment:
This parenthesis requirement duplicates appears on real RPM requires, I feel dropping this from tests may actually break real resolution 
Why are you dropping them?